src/transformStateWithClones.js

```diff
@@ -7,7 +7,36 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const stateArray = [];
+  const tempState = {};
```

Comment:
```suggestion
  const tempState = { ...state };
```
Тут сразу сделай копию